internal/httpserve/server/server.go

```
@@ -42,6 +43,8 @@ func NewServer(c config.Server, l *zap.SugaredLogger) *Server {
 
 // StartEchoServer creates and starts the echo server with configured middleware and handlers
 func (s *Server) StartEchoServer(ctx context.Context) error {
+	_ = sentry.Init(s.config.Sentry.ClientOptions())
```

Any reason not to error check this instead of ignoring it?

---

I assumed if it passed the config checks but failed due to missing variables or otherwise the best behavior would be to fail open vs. failing the error check ?

---

its a non critical, optional service for telemetry ¯\_(ツ)_/¯

---

It's non critical, but its going to silently fail. I would at least log  an error